library/src/G4LatticeLogical.cc

```diff
@@ -57,6 +57,7 @@
 //		<p> (transport momentum).
 // 20231017  E. Michaud -- Add 'AddValley(const G4ThreeVector&)'
 // 20240426  S. Zatschler -- Add explicit fallthrough statements to switch cases
+// 20240510  Add ProcessL0(G4 bool).
```

Comment:
Is this correct?  And is it really from May?  Maybe you can make the comment more narrative?  e.g., "Add function to compute L0 from other parameters".

Reply:
I changed it to "add function to compute L0 from other parameters". The date is the original date the function was added. Do you want to update the date to today's date?

Reply:
Thanks, Emile!  I didn't realize you started this that far back; sorry!